pkg/resourcesets/collector.go

```diff
@@ -235,6 +235,7 @@ func (h *ResourceHandler) filterByNameAndLabel(ctx context.Context, dr dynamic.R
 	}
 	// filter out using ResourceNameRegexp
 	if filter.ResourceNameRegexp != "" {
+		logrus.Infof("Using ResourceNameRegexp %s to filter resource names", filter.ResourceNameRegexp)
```

Comment:
Log statements should start with lowercase I think

Reply:
Added brackets.

Regarding log statements lowercase, nearly all log statements start with upper case and if we don't want that we need CI to prevent it.